### ISSUE TITLE
I'm trying to polish an assembly using consensus mode but it keeps failing to produce consensus file.

### ISSUE THREAD

**Issue by truong-phu-minh** [nanoporetech/medaka#520]
**Description**
It fails to import pyabpoa using both pip and bioconda, and can not run medaka_consensus.

**Logging**
Command line:
medaka_consensus -i document/cluster_001/4_reads.fastq -d document/cluster_001/7_final_consensus.fasta -o trycycler/cluster_001/medaka -m r104_e81_sup_g610

TF_CPP_MIN_LOG_LEVEL is set to '3'
Cannot import pyabpoa, some features may not be available.
Cannot import pyabpoa, some features may not be available.
Cannot import pyabpoa, some features may not be available.
Checking program versions
This is medaka 1.12.0
Cannot import pyabpoa, some features may not be available.
Program    Version    Required   Pass     
bcftools   1.20       1.11       True     
bgzip      1.20       1.11       True     
minimap2   2.28       2.11       True     
samtools   1.20       1.11       True     
tabix      1.20       1.11       True     
WARNING: Output trycycler/cluster_001/medaka already exists, may use old results.
Cannot import pyabpoa, some features may not be available.
[14:14:37 - MdlStrTF] Successfully removed temporary files from /var/folders/7h/h9g984v17lj3hjph_s63l11h0000gn/T/tmpkgw4zgck.
Cannot import pyabpoa, some features may not be available.
[14:14:38 - MdlStrTF] Successfully removed temporary files from /var/folders/7h/h9g984v17lj3hjph_s63l11h0000gn/T/tmpygnv01jh.
Not aligning basecalls to draft, calls_to_draft.bam exists.
Running medaka consensus
Cannot import pyabpoa, some features may not be available.
medaka_consensus: line 179: 52661 Illegal instruction: 4  medaka consensus "${CALLS2DRAFT}.bam" "${CONSENSUSPROBS}" --model "${MODEL}" --batch_size "${BATCH_SIZE}" --threads "${THREADS}" ${MAPQ_OPTS}
Failed to run medaka consensus.

**Environment (if you do not have a GPU, write No GPU):**
 - Installation method from pypi (pip install)
 - OS: OSX-64
 - medaka version 1.12.0
 - GPU model: no GPU

**Additional text**
Anyone know how to solve this issues? I ran this once and it successfully for creating minimap2, but it can not execute the medaka_consensus. I try to install medaka using bioconda channel, but it returns fail on my macbook M1.

**Comment by cjw85** [comment]
Medaka is not available as a binary distribution for ARM-based Macs. If you have managed to install medaka with a package I imagine it is because you have followed ill-advised instructions to coerce your shell into installing a x86 version of conda or Python.

The "illegal instruction" error comes from running x86 code on an ARM processor.

Medaka can only be installed from the source code on ARM-based Macbooks.

**Comment by truong-phu-minh** [comment]
Thanks for providing my error, I understand. I created a new env with ARM64 platform and I tried to install medaka from source following in my command line:
"brew install git-lfs
git clone https://github.com/nanoporetech/medaka.git
cd medaka
make install"

It seems ok until I execute make install, and this still fails and returns this:

**ERROR: Could not find a version that satisfies the requirement ont-parasail (from medaka) (from versions: none)
ERROR: No matching distribution found for ont-parasail**

I don't understand whether it is still incompatible to my ARM64 architecture? I used macOS Sonoma, I checked and this platform was compatible with OSX-ARM64 based.
